env.example

```diff
@@ -22,3 +22,6 @@ NODEMAILER_SERVER=
 # BLOG_INACTIVE_TIME is the period (days) of inactivity
 # before a blog will be considered redlisted
 BLOG_INACTIVE_TIME=360
+
+# NODEMAILER_TESTRECIPIENT is a email that will receive email messages from telescope	
+NODEMAILER_TESTRECIPIENT=adell.crist@ethereal.email
```

Comment:
I'm not sure about hardcoding the fake recipient into `env.example`. I think you can just add it to the test.

Reply:
Yeah I totally understand and was thinking the same thing, but I saw idea put up [here](https://github.com/Seneca-CDOT/telescope/pull/93). I was trying to keep it consistent with the codebase. I can still change it, what do you think?

Reply:
I added the username, password and server to the .env file because it was confidential information. You may want to check what you can do with that link. Cause I don't think stuff like emails needs to be in the .env file as recipients is hard cored in the send message function

Reply:
Okay, I've removed it from the .env file. Please take a look

Reply:
I used test recipient on the .env file because sendMessage needs a recipient to be passed for it to be tested with the expected output. Also, the reason I put it in the .env file is so anyone who is pulls down the codebase and wants to test with different email addresses can go directly to .env and not have to search the codebase to look where to find it. I could instead just remove the email and leave NODEMAILER_TESTRECIPIENT to be empty?